docs/zkapps/o1js/interact-with-mina.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Interacting With Mina
 hide_title: true
-description: How to create zero knowledge proofs and how users call zkApp methods. An account update can have proof, signature or none authorizations.
+description: How to create zero knowledge proofs and how users can call zkApp methods. An account update can have proof or signature authorizations.
```

Comment:
we have conflicting statements: proof, signature, AND none (or just proof or signature?) do we need an explainer?